tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -0,0 +1,30 @@
+package cloudprovider
+
+import (
+	"fmt"
+
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/client"
+)
+
+// CloudProviderFactory returns cloud specific interface for performing necessary functions related to creating or
+// destroying an instance.
+// The factory takes in kubeconfig of an existing OpenShift cluster and a cloud vendor specific credential file.
+// Since the credential file may contain multiple accounts and the default account name/ID varies between providers,
+// this function requires specifying the credentialAccountID of the user's credential account.
+// The resourceTrackerDir is where the 'installed information' such as created instance and security group ID is saved.
```

Comment:
You yet again did not fix this. 
`resourceTrackerDir is the directory where the windows-node-installer.json file which contains IDs of created instance and security group will be created`